modules/rollback-operator-upgrades-cli.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-operator.adoc
+:_content-type: PROCEDURE
+[id="rollback-operator-upgrades_{context}"]
+= Rolling back an Operator upgrade by using the CLI
+
+You can roll back the Operator version by using CLI commands.
+[NOTE]
+====
+This procedure can only be used for {product-title-short} release 3.74 and earlier when the `rocksdb` database is installed.
+====
+
+.Procedure
+
+. Delete the {olm} subscription by running the following command:
+* For {ocp}:
++
+[source,terminal]
+----
+$ oc -n rhacs-operator delete subscription rhacs-operator
+----
+* For Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete subscription rhacs-operator
+----
+. Delete the cluster service version (CSV) by running the following command:
+* For {ocp}:
++
+[source,terminal]
+----
+$ oc -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
+----
+* For Kubernetes:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
+----
+. Determine the previous version you want to roll back to by choosing one of the following options:
+* If the current Central instance is running, query the {product-title-short} API to get the rollback version by running the following command:
++
+[source,terminal]
+----
+$ curl -k -s -u <user>:<pass> https://<central hostname>/v1/centralhealth/upgradestatus | jq -r .upgradeStatus.forceRollbackTo
+----
+* If the current Central instance is not running, perform the following steps.
+.. Ensure the Central deployment is scaled down by running the following command:
+** For {ocp}:
++
+[source,terminal]
+----
+$ oc scale -n <central namespace> –replicas=0 deploy/central
+----
+** For Kubernetes:
++
+[source,terminal]
+----
+$ kubectl scale -n <central namespace> –replicas=0 deploy/central
+----
+.. Create a pod in your Central namespace using the following pod spec:
```

Comment:
@kylape Do they need to run a command to create the pod? If so, should we include it here?

Reply:
Yeah, I suppose I was leaving to the reader to figure out how to create the pod :smile:.  Something like 

```
cat << EOF | oc create pod -n <central namespace> -f -
apiVersion: v1
kind: Pod
metadata:
  name: get-previous-db-version
spec:
  containers:
  - name: get-previous-db-version
    image: registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:<rollback version>
    command:
    - sh
    args:
    - '-c'
    - "cat /var/lib/stackrox/.previous/migration_version.yaml | grep '^image:' | cut -f 2 -d : | tr -d ' '"
    volumeMounts:
    - name: stackrox-db
      mountPath: /var/lib/stackrox
  volumes:
  - name: stackrox-db
    persistentVolumeClaim:
      claimName: stackrox-db
EOF
```

Reply:
Alternatively we could have the reader save the pod yaml to disk and do `oc create -n <central namespace> -f pod.yaml`